aries-site/src/components/index.js

```diff
@@ -1,2 +1,3 @@
 export * from './content';
 export * from './headings';
+export { Meta } from './seo';
```

Comment:
Can this just be the following?
```suggestion
export * from './seo';
```